pkg/neg/syncers/subsets.go

```
@@ -173,13 +173,21 @@ func getSubsetPerZone(nodesPerZone map[string][]*v1.Node, totalLimit int, svcID
 	// Sort zones in increasing order of node count.
 	zoneList := sortZones(nodesPerZone)
 
+	defaultSubnet, err := utils.KeyName(networkInfo.SubnetworkURL)
```

made a comment on a previous commit. we should just store this and calculate it at initialization. if there is a problem better to throw an error at startup because it will alway cause an error